index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="index.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/df740146e4.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    
+    
+    <header class="nav">
+    <a href="#" class="logo">Portfolio</a>
+    <a href="#" class="menu"><i class="fa-solid fa-bars"></i></a>
+    </header>
+    
+    <main class="main">
```

Comment:
Great job @Fahim-Khan-P 🤝

- [ ] As stated in the project requirements, kindly align elements of this section with `display: flex`.